packages/features/portfolio/src/containers/portfolio-overview/portfolio-overview-mobile.tsx

```diff
@@ -0,0 +1,80 @@
+import { MultiDisplayChart } from '@notional-finance/mui';
+import { observer } from 'mobx-react-lite';
+import { useTotalsChart, useTotalHoldingsTable } from './hooks';
+import { usePortfolioNOTETable, useVaultHoldingsTable } from '../../hooks';
+import { Box, styled, useTheme } from '@mui/material';
+import { useAppStore } from '@notional-finance/notionable-hooks';
+import PortfolioHoldingsOverview from './containers/portfolio-holdings-overview';
+import LeverageVaultsOverview from './containers/leverage-vaults-overview';
+import LineChart from '@notional-finance/mui/lib/line-chart/line-chart';
+import moment from 'moment';
+import NOTEHoldingsOverview from './containers/note-holdings-overview';
+
+const PortfolioOverviewMobile = () => {
+  const theme = useTheme();
+  const { baseCurrency } = useAppStore();
+  const { totalHoldingsData, showTotalHoldingsTable } =
+    useTotalHoldingsTable(baseCurrency);
+  const { vaultHoldingsData, showVaultHoldingsTable } = useVaultHoldingsTable();
+  const { noteData } = usePortfolioNOTETable();
+  const { barChartData, barConfig, totalsData } = useTotalsChart(baseCurrency);
+
+  return (
+    <Box>
+      <Container>
+        {barChartData && barConfig && (
+          <MultiDisplayChart
```

Comment:
![image](https://github.com/user-attachments/assets/6604162d-bc22-45fe-b482-ebb7a835b8da)

Can we change the font size for the APY figure to match the account value figure? It the chart is just showing a month of data but the time range on desktop is much larger:
![image](https://github.com/user-attachments/assets/a6f81941-b806-4189-b306-7815a5c136e6)

Also can we match the tooltip on the desktop chart version:
![image](https://github.com/user-attachments/assets/ff087127-b0b5-4a6e-a40e-f390f6d5cad0)